src/components/Views.tsx

```diff
@@ -91,10 +108,11 @@ const ActiveListView = ({ views, removeAll, applyLayout, remove }: TActiveListVi
             </div>
             <div className='sc-views__views__content'>
                 <div className='sc-views__views__list'>
-                    {views.map((view, i) => (
+                    {sorted_views.map((view, i) => (
                         <ViewItem
+                            disabled={isDisabled(view.layout)}
                             view={view}
-                            key={view.name}
+                            key={`${view.name}-${i.toString()}`}
```

Comment:
`view.name` is set by a user, so it can be the same for more than 1 template. we need a unique key here, hence adding a stringified index for this purpose.

Reply:
I didn't get the idea of stringified index. If it's number and it's inserted in backticks, won't it be a part of the string? Just curious